src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -596,5 +596,113 @@ public void LogInternalAPITrainingSummary(string RequestId, string TrainingId, s
         }
 
         #endregion Internal AI API Events (ID Range : 4000 - 4199)
+
+        #region Runtime Events (ID Range: 5000 - 5199)
+
+        /// <summary>
+        /// Log runtime host message (Error/Critical).
+        /// </summary>
+        /// <param name="RequestId">Request id.</param>
+        /// <param name="Source">The source.</param>
+        /// <param name="SubscriptionId">Subscription id.</param>
+        /// <param name="ResourceGroup">Resource group.</param>
+        /// <param name="Resource">The resource.</param>
+        /// <param name="ExceptionType">Exception type.</param>
+        /// <param name="ExceptionDetails">Exception details.</param>
+        /// <param name="Message">The message.</param>
+        [Event(5000, Level = EventLevel.Error, Channel = EventChannel.Admin, Message = ETWMessageTemplates.LogRuntimeMessage)]
+        public void LogRuntimeLogError(string RequestId, string Source, string SubscriptionId, string ResourceGroup, string Resource, string ExceptionType, string ExceptionDetails, string Message)
```

Comment:
My guess is that this tracing will be used widely by detector code. Do they have access to RequestId? Would it possible to add a DetectorId column to the table so that we can easily filter out events from Detectors specifically?

Reply:
Ignore my comment. I see the Source is DetectorId and I see how you are passing RequestId so cool!

Reply:
No problem!